net/quagga/files/quagga.init

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006 OpenWrt.org
+
+START=60
+start() {
+	/usr/sbin/quagga.init start
```

Comment:
Now this makes no sense. Why not combine both scripts instead of having one call the other?